ZendeskSDK Urban Airship Example App/AppDelegate.m

```diff
@@ -100,7 +96,7 @@ - (void)application:(UIApplication *)application didRegisterForRemoteNotificatio
 - (void)application:(UIApplication *)application didReceiveRemoteNotification:(NSDictionary *)userInfo fetchCompletionHandler:(void (^)(UIBackgroundFetchResult))completionHandler
 {
     
-    [ZDKPushUtil handlePush:[[NSDictionary alloc] initWithDictionary:pushUtilUserInfo]
+    [ZDKPushUtil handlePush:[[NSDictionary alloc] initWithDictionary:userInfo]
              forApplication:application
```

Comment:
Did you investigate about the `userInfo` variable and what exactly happened to the `pushUtilUserInfo`?